bci-ssvep/database/results.sql

```diff
@@ -0,0 +1,3 @@
+select cast(sum(json_extract(process_results.results, '$.total_correct_itcca')) as double)/cast(sum(json_extract(process_results.results, '$.total_trials_tested')) as double) as itcca_accuracy,
+       cast(sum(json_extract(process_results.results, '$.total_correct')) as double)/cast(sum(json_extract(process_results.results, '$.total_trials_tested')) as double) as cca_accuracy
+from process_results
```

Comment:
There is supposed to be `where` filter on `processs_param_id` with :processParamId as parameter. Usually all consoles and etc. will recognize named parameter and its simpler to run (like DB Beaver)